app/components/CrashMapCard.tsx

```diff
@@ -41,9 +46,20 @@ export default function CrashMapCard({
   });
   const { mutate, loading: isMutating } = useMutation(mutation);
 
+  const hasCoordinates = !!savedLatitude && !!savedLongitude;
+
+  const hasLocation = !!locationId;
+
   return (
     <Card>
-      <Card.Header>Location</Card.Header>
+      <Card.Header>
+        Location:{" "}
+        {hasLocation ? (
+          <Link href={`/locations/${locationId}`}>{locationId}</Link>
+        ) : (
+          "unassigned"
```

Comment:
is there something you would prefer this to say instead?

Reply:
nah, this is fine for now. i kinda want different "Location" to be a heavier font weight, but we can save that for https://github.com/cityofaustin/vision-zero/pull/1676 or a follow-up to it